rawdisk/scheme/mbr.py

```diff
@@ -47,6 +56,43 @@ def part_offset(self):
     def part_type(self):
         return self.fields.part_type
 
+    def chs2lba(self, cyl, head, sect, geometry=DEFAULT_GEOMETRY):
+        """ helper for consistency check """
+        hpc, spt = geometry['HPC'], geometry['SPT']
+        return sect-1 + head*spt + cyl*hpc*spt
+
+    def get_type_label(self):
+        if self.fields.part_type in TYPES:
+            return '%s' % TYPES[self.fields.part_type]
+        else:
+            return 'unknown'
+
+    def __str__(self):
+        # calculated values
+        fields = self.fields.copy()
+        fields['chs_start_sector'] = self.chs2lba(
```

Comment:
Because you adding new functionality inside __str__ method.. if i print an object i expect those properties to be there, except for minor formatting, that way i can at least test this without comparing strings